src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -651,5 +656,10 @@
         false,
     );
 
+    doc.body.dispatchEvent(
+        new CustomEvent('ibexa-recall-autogenerate-identifier', {
+            detail: { fieldNode: global.document, shouldAutogenerateValue: true },
```

Comment:
I don't think this should be done, it will duplicate listeners attached to the input because one was added already from the autogenerate.identifier.js

Reply:
This needs to be done if we want to have this function for a non-empty identifier field. It runs the function with different value for second parameter